docs/layouts/shortcodes/cli-installation-requirements.md

```diff
@@ -0,0 +1,6 @@
+You need to have:
+
+* [PHP 5.5.9+](https://www.php.net/manual/en/install.php) with the following extensions installed: `curl`, `json`, `mbstring`, `pcre`, and `phar`.
+* [Git](https://docs.github.com/en/get-started/quickstart/set-up-git). It's the open source version control system used to manage any changes you make to your project.
+* A Bash-like shell.
+  On Windows, the best way to get Bash is through [Windows Subsystem for Linux](https://msdn.microsoft.com/en-gb/commandline/wsl/about).
```

Comment:
:hammer_and_wrench: 
I liked it as a separate paragraph before, because it's easier to skip past when I'm not on Windows